controller/uninstall_controller.go

```diff
@@ -296,11 +296,15 @@ func (c *UninstallController) uninstall() error {
 		return err
 	}
 
-	if waitForUpdate, err := c.deleteWebhookDeployment(types.LonghornAdmissionWebhookDeploymentName); err != nil || waitForUpdate {
+	if waitForUpdate, err := c.deleteDeployment(types.LonghornRecoveryBackendDeploymentName); err != nil || waitForUpdate {
 		return err
 	}
 
-	if waitForUpdate, err := c.deleteWebhookDeployment(types.LonghornConversionWebhookDeploymentName); err != nil || waitForUpdate {
+	if waitForUpdate, err := c.deleteDeployment(types.LonghornAdmissionWebhookDeploymentName); err != nil || waitForUpdate {
+		return err
+	}
+
+	if waitForUpdate, err := c.deleteDeployment(types.LonghornConversionWebhookDeploymentName); err != nil || waitForUpdate {
 		return err
 	}
```

Comment:
nit: this is good, but probably we can have a loop for these deletions because the thing is operation is the same.

Reply:
Updated. Thank you.